eng/WpfArcadeSdk/tools/ApiCompat.targets

```diff
@@ -84,7 +84,7 @@
     <!-- If DotNetTool is undefined, we default to assuming 'dotnet' is on the path -->
     <DotNetTool Condition="'$(DotNetTool)' == ''">dotnet</DotNetTool>
 
-    <_ApiCompatPath Condition="'$(MSBuildRuntimeType)' == 'core'">$(NuGetPackageRoot)\microsoft.dotnet.apicompat\$(MicrosoftDotNetApiCompatVersion)\tools\netcoreapp2.1\Microsoft.DotNet.ApiCompat.dll</_ApiCompatPath>
+    <_ApiCompatPath Condition="'$(MSBuildRuntimeType)' == 'core'">$(NuGetPackageRoot)\microsoft.dotnet.apicompat\$(MicrosoftDotNetApiCompatVersion)\tools\net10.0\Microsoft.DotNet.ApiCompat.dll</_ApiCompatPath>
```

Comment:
This seems to be the old apicompat which we delete from Arcade more than a year ago. There's definitely not a package out there with a net10.0 TFM in it.

Reply:
Odd. With this change it seems to work.